app/views/pages/about.html.erb

```diff
@@ -0,0 +1,22 @@
+<div class="container release-notes col-md-12">
+  <h1 class="page-heading">About the New Catalog</h1>
+
+  <p>Initially released on 28 April 2016, this catalog is the result of a nearly two year long project to replace our twenty year old <%= t('blacklight.voyager') %> and incorporate the best features of new types of library search technology.</p>
```

Comment:
Pretty sure it should be "twenty-year-old." I know it's been like this for a while.